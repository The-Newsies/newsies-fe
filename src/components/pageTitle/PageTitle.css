.title {
  display: flex;
  justify-content: center;
  align-items: center;
  width: 100%;
  height: 120px;
  background-color: #FFF;
  margin-top: -20px;
}

.title h1 {
  padding-top: 15px;
}