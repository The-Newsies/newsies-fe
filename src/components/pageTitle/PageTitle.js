import React from 'react';
import PropTypes from 'prop-types';

export default function PageTitle({ title }) {
  return (
    <>
      <h1>{title}</h1>
    </>
  );
}

PageTitle.propTypes = {
  title: PropTypes.string.isRequired
};
