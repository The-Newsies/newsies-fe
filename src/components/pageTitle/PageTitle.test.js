import React from 'react';
import { shallow } from 'enzyme';
import PageTitle from './PageTitle';

describe('PageTitle component', () => {
  it('renders PageTitle', () => {
    const wrapper = shallow(<PageTitle title={'title'}/>);
    expect(wrapper).toMatchSnapshot();
  });
});
