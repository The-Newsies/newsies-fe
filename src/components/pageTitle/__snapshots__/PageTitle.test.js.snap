// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageTitle component renders PageTitle 1`] = `
<Fragment>
  <h1>
    title
  </h1>
</Fragment>
`;
